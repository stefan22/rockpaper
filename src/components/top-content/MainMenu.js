import React from 'react';
import {NavLink} from 'react-router-dom';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import IconButton from '@material-ui/core/IconButton';
import hand from '../../images/handshake.png';
import '../../scss/main-menu.scss';

const MainMenu = () => (
  <section className='topmenu'>
    <div className='full-container'>
      <AppBar position="static">
        <Toolbar>
          <IconButton className='topbar-icon' color="inherit">
            <img src={hand} alt='handshake' width='960' />
          </IconButton>
          <NavLink className='nav topbar-home'
            to='/' activeClassName='is-active' exact={true}>
            <Typography variant="h6" color="inherit" className='topbar-grow'>
              Rock•Paper•Scissors
            </Typography>
          </NavLink>
        </Toolbar>
      </AppBar>
    </div>
  </section>
);



export default MainMenu;
