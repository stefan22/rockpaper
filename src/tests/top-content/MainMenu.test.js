import React from 'react';
import {shallow, mount, render} from 'enzyme';
import toJSON from 'enzyme-to-json';
import MainMenu from '../../components/top-content/MainMenu';

describe('MainMenu component', () => {
  it('should render MainMenu component correctly', () => {
    const wrapper = shallow(<MainMenu />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });


});
