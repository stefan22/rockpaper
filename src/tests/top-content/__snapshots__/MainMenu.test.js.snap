// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainMenu component should render MainMenu component correctly 1`] = `
<section
  className="topmenu"
>
  <div
    className="full-container"
  >
    <WithStyles(AppBar)
      position="static"
    >
      <WithStyles(Toolbar)>
        <WithStyles(IconButton)
          className="topbar-icon"
          color="inherit"
        >
          <img
            alt="handshake"
            src="handshake.png"
            width="960"
          />
        </WithStyles(IconButton)>
        <NavLink
          activeClassName="is-active"
          aria-current="page"
          className="nav topbar-home"
          exact={true}
          to="/"
        >
          <WithStyles(Typography)
            className="topbar-grow"
            color="inherit"
            variant="h6"
          >
            Rock•Paper•Scissors
          </WithStyles(Typography)>
        </NavLink>
      </WithStyles(Toolbar)>
    </WithStyles(AppBar)>
  </div>
</section>
`;
